third_party/nvfuser/csrc/python_frontend/python_bindings.cpp

```diff
@@ -24,7 +24,7 @@ void initNvFuserPythonBindings(PyObject* module) {
   auto nvfuser = py::handle(module).cast<py::module>();
 
   //! DataTypes supported by nvFuser in the FusionDefinition
-  py::enum_<Nvf::DataType>(nvfuser, "DataType")
+  py::enum_<Nvf::PrimDataType>(nvfuser, "DataType")
```

Comment:
If we are renaming it here, I'm surprised that I didn't see the renaming on python side? Are we passing python tests?!

I'd like to keep this renaming on the cpp side if that makes sense. I think we can get away with mapping the rename in our python wrapper, so we don't have to change a bunch of user scripts.

cc'ing @kevinstephano for opinion on whether to expose this to user.

Reply:
Yes, I checked python tests, and they are passing. The python side is not renamed. I think C++ generally doesn't care about names, so there is nothing that prevents you from exposing a `class BigAndBeautiful` as `"SmallAndUgly"` on python.